scripts/generator/graphql_generator/csharp/SDK/ShopifyError.cs.erb

```diff
@@ -0,0 +1,17 @@
+namespace <%= namespace %>.SDK {
+    using System.Collections.Generic;
+
+    public class ShopifyError {
+        public enum ErrorType {
```

Comment:
I believe `Type` might be better, so it reads `ShopifyError.Type`, 🤷‍♂️ just my opinion